util/awsutil/rds.go

```diff
@@ -37,44 +38,99 @@ func FindRdsIdentifier(config config.ServerConfig, sess *session.Session) (ident
 	return
 }
 
-func FindRdsInstance(config config.ServerConfig, sess *session.Session) (instance *rds.DBInstance, err error) {
-	var resp *rds.DescribeDBInstancesOutput
+func findRdsClusterByIdentifier(clusterIdentifier string, svc *rds.RDS) (*rds.DBCluster, error) {
```

Comment:
Side effect of the way this gets called through `FindRdsIdentifier` => `FindRdsInstance`, is that in the case of a failover it will take us up to 5 minutes to update the mapping thats cached in `FindRdsIdentifier`.

This might be confusing/unexpected, since it'd show the wrong logs in pganalyze for those 5 minutes, instead of switching over.

We could consider having a shorter, separate TTL for the cluster => instance mapping, and handle that separately?

(e.g. assuming we download logs every 30 seconds, it might be reasonable to a `DescribeDBClusters` that often, but we could keep the `DescribeDBInstances` result cached)

Reply:
No strong feelings here. This could be confusing, but it should be very rare, and I don't know that we want to block these changes on figuring out a good solution.

Reply:
Yeah, for now I'll leave this to avoid rate limiting problems.